src/interfaces/iMessageListener.ts

```diff
@@ -0,0 +1,14 @@
+import { Message } from "../models/message";
+
+/**
+ * Message listener interface.
+ */
```

Comment:
Useless comment, the name IMessageListener already implies that this is a Message Listener Interface.

Reply:
Added for consistency, for when generating TSDoc for example. This is also done in for example blocker-models.